site/app/libraries/NumberUtils.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace app\libraries;
+
+/**
+ * Class NumberUtils
+ *
+ * Utility functions for interacting with numbers
+ */
+
+class NumberUtils {
+
+    /**
+     * @param $x float The number to round
+     * @param $y float The number $x to be rounded with respect to this variable $y
+     * @return float The rounded result to the nearest multiple of $y
+     */
+    public static function roundPointValue($x, $y) {
```

Comment:
Let's use `$value` and `$precision` as the parameter names.

We can also add the float as direct type hints and not just in phpdoc string.

Reply:
@MasterOdin okay